content/brazilian-portuguese/web-basics/activity-1.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Atividade 1 – Familiarize-se com o editor online"
+date: 2019-09-03T17:53:41-07:00
+draft: false
+weight: 4
+---
+
+## Atividade 1 – Criando um site básico
+
+Agora é hora de começar a construir um site básico. Para concluir esta atividade, você precisará concluir as seguintes etapas:
+
+1. Abra o editor web (clique no botão "Try it yourself!" abaixo)
+2. Use as tags de cabeçalho para adicionar um título e as tags de parágrafo para escrever um parágrafo sobre Benji
+3. Se tiver tempo: escreva uma lista das guloseimas favoritas de Benji: peru, manteiga de amendoim e queijo ralado
+
+<a class="my-2 mx-4 btn btn-info" href="https://codepen.io/Sunny-Dee/pen/exxyYL" target="_blank">Try it yourself!</a>
+
+Use este gráfico de referência para testar algumas tags HTML por conta própria.
+
+Etiqueta | Descrição | Exemplo
+---|--------------|----------
+`<html></html>`                         | Descreve um documento HTML. Qualquer site deve começar com a tag de abertura <html> e terminar com a tag de fechamento </html>. | Veja o exemplo do Codepen acima.
+`<!-- -->`                              | Descreve um comentário. Os comentários devem ser vistos por outros programadores, mas não são exibidos no site. |`<!-- Esta linha não será exibida no navegador -->`
+`<head></head>`                         | Contém todos os elementos HTML que fornecem informações ao navegador para exibir o site corretamente. Os elementos HTML entre as tags <head> e </head> não são exibidos. | Veja o exemplo da Code Pen acima
+`<title></title>`                       | Contém o título do site a ser usado para nomear a aba do navegador. | &lt;head&gt;<br>&lt;título&gt; Meu site &lt;/title&gt;<br>&lt;/head&gt;
```

Comment:
the <title> tag example is in Portuguese (&lt;título&gt; Meu site &lt;/title&gt;), while the rest are in English.